docs/reference/modules/cluster/allocation_awareness.asciidoc

```diff
@@ -19,15 +19,16 @@ allocated in each location. If the number of nodes in each location is
 unbalanced and there are a lot of replicas, replica shards might be left
 unassigned.
 
+TIP: Learn more about <<high-availability-cluster-design-large-clusters,designing resilient clusters>>.
+
 [[enabling-awareness]]
 ===== Enabling shard allocation awareness
 
 To enable shard allocation awareness:
 
-. Specify the location of each node with a custom node attribute. For example,
-if you want Elasticsearch to distribute shards across different racks, you might
-set an awareness attribute called `rack_id` in each node's `elasticsearch.yml`
-config file.
+. In each node's `elasticsearch.yml` config file, specify the location of each node 
```

Comment:
You can specify the location of each node with a custom node attribute by setting a value in `elasticsearch.yml` _or_ using the `-E` command line argument. I agree that it'd be good to mention that the setting goes in `elasticsearch.yml`, but the first sentence in this paragraph should really apply to both methods.

Reply:
thanks @DaveCTurner! I've restructured this step to take your feedback into consideration: 

<img width="711" alt="image" src="https://github.com/elastic/elasticsearch/assets/58563081/bf64449f-8ba9-47ea-aa0c-ecb545fe5497">